server/lambda/etsToSbc/index.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/no-unresolved */
+const { MongoClient } = require('mongodb');
+const { sendEtsToSBC } = require('custom_modules/sbc-phac-ets');
+const readAwsRdsCA = require('custom_modules/certificates/read-aws-rds');
+
+
+/* eslint-disable no-console */
+
+const dbConnectionOptions = (server) => ({
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+  ssl: server !== 'mongodb',
+  sslValidate: server !== 'mongodb',
+  sslCA: server === 'mongodb' ? undefined : [readAwsRdsCA()],
+});
+
+const dbConnectionAndCollections = async (collections) => {
```

Comment:
This function is declared identically in both `index.js` files. Suggest we move this into the shared layer.

Reply:
Good catch, updating...